src/main/kotlin/app/revanced/patches/music/audio/codecs/fingerprints/CodecsLockFingerprint.kt

```diff
@@ -12,17 +12,14 @@ import org.jf.dexlib2.Opcode
 
 @Name("codec-lock-fingerprint")
 @MatchingMethod(
-    "Labwj;",
+    "Lacfr;",
     "a",
 )
 @FuzzyPatternScanMethod(2) // FIXME: Test this threshold and find the best value.
 @CodecsUnlockCompatibility
 @Version("0.0.1")
 object CodecsLockFingerprint : MethodFingerprint(
-    "L", AccessFlags.PUBLIC or AccessFlags.STATIC, listOf("L", "L", "L", "L"), listOf(
-        Opcode.INVOKE_VIRTUAL,
-        Opcode.MOVE_RESULT_OBJECT,
-        Opcode.CHECK_CAST,
+    "L", AccessFlags.PUBLIC or AccessFlags.STATIC, null, listOf(
```

Comment:
Does this fingerprint still work on older versions? Just making sure.
If not, remove compatibility for older versions or make it a new/separate fingerprint.

Reply:
> Does this fingerprint still work on older versions
yes thats why parameters and some opcodes are removed so it works in both

Reply:
Alright!